openlibrary/templates/home/stats.html

```diff
@@ -15,13 +15,13 @@ <h2 class="inline">$_('Around the Library')</h2>
         </div>
 
         <div class="statschart">
-          <div class="chartShow" onClick="window.location.href='/recentchanges';" title="$_('How many new Open Library members have we welcomed?')">
+          <div class="chartShow" onClick="window.location.href='//archive.org/stats';" title="$_('How many new Open Library members have we welcomed?')">
             <div id="members-graph" style="width:150px;height:60px;"></div>
             <a href="/recentchanges" title="$_('How many new Open Library members have we welcomed?')"><span class="ticks">$:commify(stats["members"].get_summary(28))</span><span class="label">$_('New Members')</span></a>
           </div>
         </div>
         <div class="statschart">
-          <div class="chartShow" onClick="window.location.href='/recentchanges';" title="$_('People are constantly updating the catalog')">
+          <div class="chartShow" onClick="window.location.href='//archive.org/stats';" title="$_('People are constantly updating the catalog')">
```

Comment:
I believe the consensus on the issue was to just change the link for "members"; `/recentchanges` makes sense for the catalog edits graph.
```suggestion
          <div class="chartShow" onClick="window.location.href='/recentchanges';" title="$_('People are constantly updating the catalog')">
```